apps/src/code-studio/components/progress/ProgressDot.jsx

```diff
@@ -282,7 +295,12 @@ export const ProgressDot = Radium(React.createClass({
          ]}
       >
         {(iconClassFromIconType[level.icon] && !isPeerReview) ?
-          <span data-tip data-for={this.tooltipId()} aria-describedby={this.tooltipId()} style={{borderColor: "#ff0000", borderWidth: 2}}>
+          <div
+            data-tip
+            data-for={this.tooltipId()}
+            aria-describedby={this.tooltipId()}
+            style={styles.iconBasedDot}
+          >
```

Comment:
This change has the chance of affecting things not behind the progress redesign flag. I changed it from a span to an inline-block div so that our tooltip shows up in the right place. I did a bunch of manual checking of things, and then will depend on UI test coverage.